docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/install.md

```diff
@@ -113,54 +106,45 @@ playback speed.
 Use the following steps to install Palette VerteX.
 
-1.  Log in to your vCenter environment.
+1.  In your terminal, use the following command template to SSH into the airgap support VM. Enter the path to your
+    private SSH key, your username, and the IP or domain of the airgap support VM. The default username is `ubuntu`.
 
-2.  Create a vSphere VM and Template folder with the name `spectro-templates`. Ensure this folder is accessible by the
-    user account you will use to deploy the airgap VerteX installation.
+    ```shell
+    ssh -i </path/to/private/key> ubuntu@<vm-ip-or-domain>
+    ```
 
-3.  Use the URL below to import the Operating System and Kubernetes distribution OVA required for the install. Place the
-    OVA in the `spectro-templates` folder. Refer to the
-    [Import Items to a Content Library](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-vm-administration/GUID-B413FBAE-8FCB-4598-A3C2-8B6DDA772D5C.html?hWord=N4IghgNiBcIJYFsAOB7ATgFwAQYKbIjDwGcQBfIA)
-    guide for information about importing an OVA in vCenter.
+    Consider the following command example for reference.
 
-    ```url
-    https://vmwaregoldenimage.s3.amazonaws.com/u-2204-0-k-12813-0.ova
+    ```shell
+    ssh -i /docs/ssh-private-key.pem ubuntu@vertex.example.com
     ```
 
-4.  Append an `r_` prefix to the OVA name and remove the `.ova` suffix after the import. For example, the final output
-    should look like `r_u-2204-0-k-12813-0`. This naming convention is required for the install process to identify the
-    OVA. Refer to the [Supplement Packs](../../airgap/supplemental-packs.md#additional-ovas) page for a list of
-    additional OVAs you can download and upload to your vCenter environment.
-
     :::tip
 
-    You can also use the **Deploy OVF Template** wizard in vSphere to make the OVA available in the `spectro-templates`
-    folder. Append the `r_` prefix, and remove the `.ova` suffix when assigning a name and target location. You can
-    terminate the deployment after the OVA is available in the `spectro-templates` folder. Refer to the
-    [Deploy an OVF or OVA Template](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-vm-administration/GUID-AFEDC48B-C96F-4088-9C1F-4F0A30E965DE.html)
-    guide for more information about deploying an OVA in vCenter.
+    Alternatively, you can use any Linux x86-64 system that has the Docker daemon and Palette CLI installed, as well as
+    connectivity to the VMware vSphere environment where Palette VerteX will be deployed.
 
     :::
 
-5.  Open a terminal window and invoke the Palette CLI by using the `ec` command to install the enterprise cluster. The
-    interactive CLI prompts you for configuration details and then initiates the installation. For more information
-    about the `ec` subcommand, refer to [Palette Commands](../../../../automation/palette-cli/commands/ec.md).
+2.  Invoke the Palette CLI by using the `ec` command to install the enterprise cluster. The interactive CLI prompts you
```

Comment:
I think we need to mention the issues with the Palette CLI existing in a different location when using the generic OVA.

Reply:
:white_check_mark: